### PR TITLE
chore: update pages metadata

### PR DESCRIPTION
**Pull request by DiogoSoaress**
## What it solves

Updates [https://safe.global](url) pages metadata based on the SEO reviewed doc https://www.notion.so/safe-global/Meta-descriptions-safe-global-a6bb06736a6d4c348dcf754a1a4b7e45#c84aea5d08b64cae87a0ee990fad9409

### CODE REVIEW

File: src/content/core.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 [
   {
-    "pageTitle": "Safe Core – Full stack account abstraction",
+    "pageTitle": "Safe{Core} — Modular Ethereum Smart Account Infrastructure | Safe",
+    "description": "Embrace limitless possibilities with Safe{Core}'s modular smart account infrastructure, tailored for all account abstraction use cases, including ERC-4337.",
```
Comment (iamacook) [inline src/content/core.json:4]: Should "smart account" be capitalised?

```suggestion
    "description": "Embrace limitless possibilities with Safe{Core}'s modular Smart Account infrastructure, tailored for all account abstraction use cases, including ERC-4337.",
```

File: src/content/careers.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
 [
   {
-    "pageTitle": "Jobs at Safe",
+    "pageTitle": "Join Our Team — Safe Careers in Smart Account Innovation | Safe",
+    "description": "Explore a career with Safe and be a part of pioneering Smart Account technology. We're hiring! Join us in shaping the future of onchain innovation.",
```
Comment (iamacook) [inline src/content/careers.json:4]: This sounds a bit strange - I'd move "We're hiring!" to the beginning but I understand that this is likely SEO-focused.

File: src/content/ecosystem.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 [
   {
-    "pageTitle": "Safe – Ecosystem",
-    "description": "Explore the Safe Ecosystem",
+    "pageTitle": "Safe Ecosystem — Powering Web3 Projects from DAOs to DeFi | Safe",
+    "description": "Explore the vast Safe Ecosystem, empowering over 200 onchain projects. From innovative DAO tooling to dynamic DeFi solutions, discover how Safe drives web3 forward.",
```
Comment (iamacook) [inline src/content/ecosystem.json:4]:
```suggestion
    "description": "Explore the vast Safe Ecosystem, empowering over 200 onchain projects. From innovative DAO tooling to dynamic DeFi solutions, discover how Safe drives Web3 forward.",
```

File: src/content/wallet.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
 [
   {
-    "pageTitle": "Safe Wallet",
-    "description": "Self-custodial multisig wallet for the Ethereum ecosystem",
+    "pageTitle": "Safe{Wallet} — Secure Smart Account Management on Ethereum | Safe",
+    "description": "Explore Safe, the leading Ethereum multisig wallet for secure and efficient smart account management. Experience advanced security and user-friendly features.",
```
Comment (iamacook) [inline src/content/wallet.json:4]: As before regarding "smart account":

```suggestion
    "description": "Explore Safe, the leading Ethereum multisig wallet for secure and efficient Smart Account management. Experience advanced security and user-friendly features.",
```

File: src/content/careers.json (hunk 2 above)
Comment (DiogoSoaress) [reply src/content/careers.json:4]: Yes, I guess so. I will leave a comment in the Notion doc just to be sure 👍

**Review by iamacook — APPROVED** [review]

File: src/content/careers.json (hunk 2 above)
Comment (DiogoSoaress) [reply src/content/careers.json]: Changed it, WDYT? From an SEO point of view it is the same so I followed your suggestion.

Comment (iamacook) [reply src/content/careers.json]: Reads much better now.

**Pull request merged**